services/production-tools/templates/vault-secrets.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: ricoberger.de/v1alpha1
+kind: VaultSecret
+metadata:
```

Comment:
Nit: For consistency, also add the labels stanza here similar to pull-secret below.